modules/programs/feh.nix

```
@@ -27,6 +27,8 @@ in {
   options.programs.feh = {
     enable = mkEnableOption "feh - a fast and light image viewer";
 
+    package = mkPackageOption pkgs "feh";
```

```suggestion
    package = mkPackageOption pkgs "feh" {};
```

Sorry

---

probably needs the space to make the formatter work

```suggestion
    package = mkPackageOption pkgs "feh" { };
```